llm_applications_library/llm/generators/schema.py

```
@@ -115,12 +117,24 @@ class Model(StrEnum):
+    """Reasoning effort levels for OpenAI reasoning models (o1, o3, GPT-5, etc.)"""
+
+    NONE = "none"
+    MINIMAL = "minimal"
+    LOW = "low"
+    MEDIUM = "medium"
+    HIGH = "high"
+    XHIGH = "xhigh"
+
+
 class OpenAIGenerationConfig(BaseModel):
     """OpenAI API用の生成設定（全ての有効なパラメーターを含む）"""
```

Ruff (0.14.13)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)